### PR TITLE
Fixing minor typo

### PR DESCRIPTION
**Pull request by sapenov**
Fixing some minor typos and also tweaking install to add spark

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,14 +72,14 @@ See official pip documentation for examples and explanations: https://pip.pypa.i
 
 ```sh
 cd /path/to/evidently_repo
-pip install -e ".[dev]"
+pip install -e ".[dev,spark]"
```
Comment (DimaAmega) [inline CONTRIBUTING.md:75]: I don't think we need to have `spark` extra here. I mean, yes, we have a few extras but only `dev` is useful for like contributing

Comment (sapenov) [reply CONTRIBUTING.md:75]: Ok, I'll take it out. 
In my environment, the pytest fails without it.

Comment (DimaAmega) [reply CONTRIBUTING.md:75]: > Ok, I'll take it out. In my environment, the pytest fails without it.

Hm... Okay
I think we have to improve our docs about how to run tests :)
Thanks for info about it

**Review by DimaAmega — APPROVED** [review]

**Comment by DimaAmega** [comment]
@sapenov Thanks for this fix! You have attentive eyes 👀😊

**Pull request merged**